source/renderer/app/stores/NetworkStatusStore.js

```diff
@@ -145,6 +149,8 @@ export default class NetworkStatusStore extends Store {
     this._checkDiskSpace();
 
     this._getStateDirectoryPath();
+
+    this.actions.networkStatus.logStateSnapshot.listen(this._logStateSnapshot);
```

Comment:
@DeeJayElly please move this line to the top after this one: https://github.com/input-output-hk/daedalus/blob/6d96fd94a98ce42c03380307563b4003d6ef6fb7/source/renderer/app/stores/NetworkStatusStore.js#L112 🙏

Reply:
Done